packages/lib-classifier/src/plugins/drawingTools/models/marks/Line/Line.js

```diff
@@ -73,10 +73,15 @@ const LineModel = types
       self.y2 = y2
     }
 
+    function onPointerUp(event, callback) {
```

Comment:
This can be added to the generic Mark model, if it's the same for all Marks.

Reply:
It's not though. `TranscriptionLine` doesn't call the callback and I'd prefer to have each mark have its own ability to be flexible. Once we migrate all the drawing tools, then we can consider abstraction?